dart/lib/src/sentry.dart

```diff
@@ -109,8 +101,28 @@ class Sentry {
     hub.close();
 
     // execute integrations after hub being enabled
-    for (final integration in options.integrations) {
-      await integration(HubAdapter(), options);
+    if (appRunner != null) {
+      var runIntegrationsAndAppRunner = () async {
+        final integrations = options.integrations
+            .where((i) => !(i is RunZonedGuardedIntegration));
```

Comment:
missing .toList()

Reply:
why `toList()`?
should we just let it enumerate here instead of creating a list?

Reply:
to do this: https://github.com/getsentry/sentry-dart/pull/280#discussion_r563739005
`.toList()` is needed

or the method param should be an `Iterable` which is also fine as `List` extends `Iterable` but not the other way around